website/blog/2022-04-08-adding-fonts.md

```diff
@@ -0,0 +1,195 @@
+import './SourceSansPro-BoldItalic.woff2';
+import './SourceSansPro-Italic.woff2';
+import './SourceSansPro-Regular.woff2';
+import './SourceSansPro-Light.woff2';
+import './SourceSansPro-LightItalic.woff2';
+
+// NotoSerif WOFF
+import './NotoSerif-Bold.woff';
+import './NotoSerif-BoldItalic.woff';
+import './NotoSerif-Italic.woff';
+import './NotoSerif-Regular.woff';
+
+// NotoSerif WOFF2
+import './NotoSerif-Bold.woff2';
+import './NotoSerif-BoldItalic.woff2';
+import './NotoSerif-Italic.woff2';
+import './NotoSerif-Regular.woff2';
+```
+
+To add these fonts as your base font and secondary font, go to the global manifest located in **_/src/Blocks_** and add the following inside `globalVariables`:
+```json
+"globalVariables": {
+	//...
+	"baseFont": "SourceSansPro",
+	"secondaryFont": "NotoSerif",
+	//...
+}
+```
+
+Next, you can create a new file called **__typography.scss_** inside your **_/assets/styles/parts/utils/_** folder and add the following:
+
+```scss
+@include font-face(global-settings(baseFont), 'SourceSansPro-Light', 300);
+@include font-face(global-settings(baseFont), 'SourceSansPro-LightItalic', 300, italic);
+@include font-face(global-settings(baseFont), 'SourceSansPro-Regular', 400);
+@include font-face(global-settings(baseFont), 'SourceSansPro-Italic', 400, italic);
+@include font-face(global-settings(baseFont), 'SourceSansPro-Bold', 700);
+@include font-face(global-settings(baseFont), 'SourceSansPro-BoldItalic', 700, italic);
+
+@include font-face(global-settings(secondaryFont), 'NotoSerif-Bold', 700);
+@include font-face(global-settings(secondaryFont), 'NotoSerif-BoldItalic', 700, italic);
+@include font-face(global-settings(secondaryFont), 'NotoSerif-Italic', 400, italic);
+@include font-face(global-settings(secondaryFont), 'NotoSerif-Regular', 400);
+```
+
+Because this is a new file, we have to include it. You can do that inside **_/assets/styles/parts/_shared.scss_** file:
+```scss
+// Project specific.
+@import 'utils/shared-variables';
+@import 'utils/typography';
+```
+
+Run `npm start` to rebuild your **_public_** folder and assets. If you did everything correctly, your build should pass and you will see your fonts inside the **_public_** folder.
+
+### Using only one font in a block
+
+There are multiple ways of using fonts in a block. The simplest example is if you have only one font you want to use for that specific block. In this case, we want Heading Block to only use NotoSerif.
+
+To make our secondary font available for use, first we need to define it as a variable. We can do that in **_/assets/styles/parts/utils/_shared-variables.scss_**. Here we can see that the base font is already defined, so all we need to do is add our secondary font below.
+
+To make things a bit more complicated, we may also want to rename `--global-font-family` CSS variable to `--base-font-family`. Just don't forget to search/replace this new variable name across your project! Once we're done, it should look like this:
```

Comment:
> To make things a bit more complicated

Truly a great motivation for doing anything 😂

Reply:
```suggestion
To make things a bit more consistent, we may also want to rename `--global-font-family` CSS variable to `--base-font-family`. Just don't forget to search/replace this new variable name across your project! Once we're done, it should look like this:
```

Reply:
Oh, that's the word I've been looking for! 😄  I didn't like how it sounded, since it's not THAT complicated, but couldn't think of anything better at the moment 😅